### DISCUSSION TITLE
订阅功能问题

### DISCUSSION THREAD

**Discussion by Leno709394** [kkkgo/PaoPaoGateWay#17] [问和答]
订阅功能，不能加载我订阅里的自动测速选线路规则。 把我订阅里的内容复制到ymal。用加载ymal的模式就能正确加载，如下图功能。
因为我节点多。便宜货梯子。不稳定。非常需要自动测速切换线路功能

![image](https://github.com/kkkgo/PaoPaoGateWay/assets/104353759/8c22058c-f879-4212-af05-48468c7243de)

**Comment by kkkgo** [comment]
跟订阅功能应该无关，如果你开了`fast_node=yes`自动测速功能，那么会忽略加载除纯节点外所有规则信息，仅使用全局模式测速纯节点。

**Comment by kkkgo** [comment]
yaml和suburl模式的唯一区别是suburl使用单独的subtime更新设置，而yaml使用全局的sleeptime更新设置。

**Reply by Leno709394** [reply]
弄好，谢谢，的确是fast那个设置问题

**Reply by kkkgo** [reply]
如果没特别复杂的测速规则要弄的话，仅使用fast_node功能一般来说是可以满足自动测速选线路的需求的。
